### ISSUE TITLE
Teach colourpicker to understand colour names

### ISSUE THREAD

**Issue by AleksueiR** [fgpv-vpgf/fgpv-vpgf#864]
Keyboard users should be able to enter nice colour names when importing file-based layers.

Some colour names: https://www.wikiwand.com/en/List_of_colors:_A%E2%80%93F

Related #770

**Comment by JahidAhmed** [comment]
Working with the following library, https://github.com/mbjordan/Colors, we are able to allow users to input the 150+ HTML standard colour names such as _Dark Salmon_ and _Forest Green_. However, many of the colours in the link provided above are not allowed. Should I look for other libraries allowing a broader range of colours?

**Comment by JahidAhmed** [comment]
Also, are we going to be handling the case where the input is a colour name but in French? For example, if the user enters _rouge_.

**Comment by AleksueiR** [comment]
Yes, we do need to handle input in French.

I think 150 colours will do, but not sure why you need a separate library. I was thinking of just intercepting the input on the colour-picker field, checking against the list of colours, and passing the hex code the minicolors library in the case of a match.

**Comment by james-rae** [comment]
I suggested the library :'D
Surprised we would hardcode our own list when it's already done for us, but either way it's not hard (and with bi-lingual requirement, might be easier to just maintain our own simple JSON mapping)

**Comment by AleksueiR** [comment]
That library does extra things we don't need, and we just need a list of colour/name pairs. I agree, hardcoding stuff in is not the best, but since we need both English and French names, it seemed like a better way.

An alternative would be to get a JS library (more like a dictionary really) with just English colour names (there are many English-only ones: [1500-color-names](https://github.com/deepblue129/1500-color-names), [colornames](https://github.com/timoxley/colornames), [color-name](https://github.com/colorjs/color-name)), and maintain our own, hardcoded list with French names. If you can find a JS lib with French colour names as well, we can use it instead of the hardcoded list. 

Thoughts?

**Comment by JahidAhmed** [comment]
I came across colornames and color-name previously and didn't think they would be the best for our use, especially colornames since majority of colors involve numbers such as _blue 2_, _blue 3_, etc. which isn't as user friendly. I was thinking of using 1500-color-names instead as you pointed out, it seems like a good one.

As for French, there doesn't seem to be a libary. I'll try some more to see if I can find something otherwise I can hardcode those using proper names I can find online.

Is that fine?

**Comment by JahidAhmed** [comment]
I came across a fairly extensive list of color names in French: https://fr.wikipedia.org/wiki/Liste_de_noms_de_couleur.

Is it okay to go ahead and hardcode using this list?

The list will however need to be reviewed / verified.
